pycti/entities/opencti_settings.py

```diff
@@ -0,0 +1,378 @@
+
+            platform_modules {
+                id, enable, warning
+            }
+
+            platform_providers {
+                name, type, strategy, provider
+            }
+
+            platform_user_statuses {
+                status, message
+            }
+
+            platform_theme
+            platform_theme_dark_background
+            platform_theme_dark_paper
+            platform_theme_dark_nav
+            platform_theme_dark_primary
+            platform_theme_dark_secondary
+            platform_theme_dark_accent
+            platform_theme_dark_logo
+            platform_theme_dark_logo_collapsed
+            platform_theme_dark_logo_login
+            platform_theme_light_background
+            platform_theme_light_paper
+            platform_theme_light_nav
+            platform_theme_light_primary
+            platform_theme_light_secondary
+            platform_theme_light_accent
+            platform_theme_light_logo
+            platform_theme_light_logo_collapsed
+            platform_theme_light_logo_login
+            platform_map_tile_server_dark
+            platform_map_tile_server_light
+
+            platform_openbas_url
+            platform_openbas_disable_display
+
+            platform_openerm_url
+            platform_openmtd_url
+
+            platform_ai_enabled
+            platform_ai_type
+            platform_ai_model
+            platform_ai_has_token
+
+            platform_login_message
+            platform_consent_message
+            platform_consent_confirm_text
+
+            platform_banner_text
+            platform_banner_level
+
+            platform_session_idle_timeout
+            platform_session_timeout
+
+            platform_whitemark
+            platform_demo
+            platform_reference_attachment
+
+            platform_feature_flags {
+                id, enable, warning
+            }
+
+            platform_critical_alerts {
+                message, type
+                details {
+                    groups {
+                        id, name, description
+                    }
+                }
+            }
+
+            platform_trash_enabled
+
+            platform_protected_sensitive_config {
+                enabled
+                markings {
+                    enabled, protected_ids
+                }
+                groups {
+                    enabled, protected_ids
+                }
+                roles {
+                    enabled, protected_ids
+                }
+                rules {
+                    enabled, protected_ids
+                }
+                ce_ee_toggle {
+                    enabled, protected_ids
+                }
+                file_indexing {
+                    enabled, protected_ids
+                }
+                platform_organization {
+                    enabled, protected_ids
+                }
+            }
+
+            created_at
+            updated_at
+            enterprise_edition
```

Comment:
`enterprise_edition` has been removed in OpenCTI 6.5 freshly released, replaced by `platform_enterprise_edition`